### DISCUSSION TITLE
Github pages dont work

### DISCUSSION THREAD

**Discussion by lascu23** [community/community#69438] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I try to deploy a site, but it returns 404 error. I tried everything but it doesnt work. The repo is "airlineDemo". Thanks for any help.

**Comment by seyLu** [comment]
#### Hi lascu23,

Github Pages is used to host static websites -- just frontend, no backend, no database.

I've noticed you're using Express.js, which is a backend web js framework. You can visit this [repo](https://github.com/ripienaar/free-for-dev), try to find a hosting platform that supports Node.js, and use that instead of Github Pages.

If you're insistent on deploying to Github Pages, instead of using express, use vanilla html/css/js or use a static site web builder. Some examples are: 
- [Hugo](https://gohugo.io/)
- [Jekyll](https://jekyllrb.com/)
- [Astro](https://astro.build/)
- [Eleventy](https://www.11ty.dev/)
